rush-plugins/rush-archive-project-plugin/src/commands/archive.ts

```diff
@@ -55,8 +58,16 @@ ${consumingProjectNames.join(", ")}`);
     spinner = ora('Attempting to create a git checkpoint branch');
     const branchName: string = getCheckpointBranch(rushConfiguration.rushJsonFolder,packageName);
     spinner.succeed(`Git Checkpoint created at branch: ${branchName}`);
+    // Prompt user to push the newly created branch
+    const { pushBranch } = await inquirer.prompt([
```

Comment:
Beside used as a command line tool, this `archive` function may be also invoked as Node API. So, if introduced a new prompt here, it hangs when used as Node API usage.


I can see there are different solutions:

1. move the prompt in `cli.ts` and do the rest of logic
2. Or, Add a `shouldPushGitCheckpoint` option for `archive` function, and skip the prompt here if the option passed

Reply:
I added the shouldPushGitCheckpoint option, is there a node api call I need to modify as well?